source/help/getting-started/searching.rst

```diff
@@ -0,0 +1,116 @@
+Searching
+=========
+
+--------------
+
+The search box in Mattermost returns search results from any channel of
+which you’re a member.
+
+-  Searching multiple words returns results with any one of the words
+   listed.
+-  When results appear, click the **Jump** link to view that post in the
+   channel archive.
+-  You can expand the search results window by clicking the “Expand”
+   button in the top right corner.
+-  Use search modifiers, like ``from:dave``, to return results only from
+   certain people or in certain channels (see below)
+
+Like many search engines, highly common words like ``the``, ``which``,
+``are`` (known as “stop words”), as well as two-letter and one-letter
+search terms, are not shown in search because they typically return too
+may results.
+
+Search Modifiers
+----------------
+
+From: and In:
+^^^^^^^^^^^^^
+
+Use ``from:`` to find posts from specific users and ``in:`` to find
+posts in specific channels.
+
+-  For example: Searching ``Mattermost in:town-square`` only returns
+   messages in Town Square that contain ``Mattermost``.
+
+“Quotation Marks”
+^^^^^^^^^^^^^^^^^
+
+Use quotation marks to return search results for exact terms.
+
+-  For example: Searching ``"Mattermost website"`` returns messages
+   containing the entire phrase ``Mattermost website`` and not messages
+   containing only ``Mattermost`` or ``website``.
+
+Wildcard\*
+^^^^^^^^^^
+
+Use the ``*`` character for wildcard searches that match within words.
+
+-  For example: Searching for ``rea*`` brings back messages containing
+   ``reach``, ``reason`` and other words starting with ``rea``.
+
+Hashtags
+--------
+
+Hashtags are searchable labels for posts. Search for any posts
+containing a hashtag by clicking the hashtag in an existing post or
+typing the hashtag with the pound symbol into the search bar. Create
+hashtags in any post by using the pound sign ``#`` followed by
+alphanumeric or other unicode characters.
+
+Valid hashtags:
+
+- Do not start with a number.
+- Are at least 3 characters long, not including the ``#``.
+- Are made up of alphanumeric or other unicode characters.
+- May contain dots, dashes or underscores.
+
+Examples: ``#bug``, ``#marketing``, ``#v2.1``, ``#user_testing``,
```

Comment:
#v2.1 is not a valid hashtag currently. ie it does not create a link when posted